srcpkgs/caps/template

```diff
@@ -0,0 +1,13 @@
+# Template file for 'caps'
+pkgname=caps
+version=0.9.26
+revision=1
+build_style=gnu-makefile
+makedepends="ladspa-sdk"
+short_desc="The LADSPA C* Audio Plugin Suite"
+maintainer="Olga Ustuzhanina <me@laserbat.pw>"
+license="GPL-3.0-or-later"
+homepage="http://quitte.de/dsp/caps.html"
+distfiles="http://quitte.de/dsp/${pkgname}_${version}.tar.bz2"
+checksum=e7496c5bce05abebe3dcb635926153bbb58a9337a6e423f048d3b61d8a4f98c9
+make_use_env=yes
```

Comment:
move `make_use_env=yes` after build style and add `make_build_args='CC=$(CXX)'` in the line after it.